ERROR - 2022-04-03 19:59:03 --> Error connecting to the database: php_network_getaddresses: getaddrinfo failed: Name or service not known
CRITICAL - 2022-04-03 19:59:03 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo failed: Name or service not known
#0 /app/myapp/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /app/myapp/system/Database/BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#2 /app/myapp/system/Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 /app/myapp/system/BaseModel.php(584): CodeIgniter\Model->doFindAll()
#4 /app/myapp/app/Models/ConciertoModel.php(64): CodeIgniter\BaseModel->findAll()
#5 /app/myapp/app/Controllers/ConciertoCrud.php(15): App\Models\ConciertoModel->todEs()
#6 /app/myapp/system/CodeIgniter.php(832): App\Controllers\ConciertoCrud->index()
#7 /app/myapp/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController()
#8 /app/myapp/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest()
#9 /app/myapp/public/index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-04-03 20:46:04 --> mysqli_connect(): php_network_getaddresses: getaddrinfo failed: Name or service not known
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 /app/myapp/app/Views/base/lista.php(18): mysqli_connect()
#2 /app/myapp/system/View/View.php(210): include('/app/myapp/app/...')
#3 /app/myapp/system/View/View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 /app/myapp/system/Common.php(1089): CodeIgniter\View\View->render()
#5 /app/myapp/app/Controllers/NavegarBaseController.php(8): view()
#6 /app/myapp/system/CodeIgniter.php(832): App\Controllers\NavegarBaseController->index()
#7 /app/myapp/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController()
#8 /app/myapp/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest()
#9 /app/myapp/public/index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-04-03 20:48:03 --> mysqli_connect(): php_network_getaddresses: getaddrinfo failed: Name or service not known
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 /app/myapp/app/Views/base/lista.php(18): mysqli_connect()
#2 /app/myapp/system/View/View.php(210): include('/app/myapp/app/...')
#3 /app/myapp/system/View/View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 /app/myapp/system/Common.php(1089): CodeIgniter\View\View->render()
#5 /app/myapp/app/Controllers/NavegarBaseController.php(8): view()
#6 /app/myapp/system/CodeIgniter.php(832): App\Controllers\NavegarBaseController->index()
#7 /app/myapp/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController()
#8 /app/myapp/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest()
#9 /app/myapp/public/index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
